src/CECILL-1.1.xml

```diff
@@ -0,0 +1,552 @@
+                </li>
+            </list>
+            <p>These expressions may be used both in singular and plural form.</p>
+        </li>
+        <li>
+            <p>Article 2 - PURPOSE</p>
+            <p>The purpose of the Agreement is to enable the Licensor to grant the Licensee a free, non-exclusive,
+             transferable and worldwide License for the Software as set forth in Article 5 hereinafter for the
+             whole term of protection of the rights over said Software.</p>
+        </li>
+        <li>
+            <p>Article 3 - ACCEPTANCE</p>
+            <list>
+                <li>
+                    <b>3.1.</b>
+                    <p>The Licensee shall be deemed as having accepted the terms and conditions of this Agreement by the
+                     occurrence of the first of the following events:</p>
+                     <list>
+                        <li>
+                            <b>(i)</b>
+                            <p>loading the Software by any or all means, notably, by downloading from a remote server, or by
+                             loading from a physical medium;</p>
+                        </li>
+                        <li>
+                            <b>(ii)</b>
+                            <p>the first time the Licensee exercises any of the rights granted hereunder.</p>
+                        </li>
+                    </list>
+                </li>
+                <li>
+                    <b>3.2.</b>
+                    <p>One copy of the Agreement, containing a notice relating to the specific nature of the Software,
+                     to the limited warranty, and to the limitation to use by experienced users has been provided
+                     to the Licensee prior to its acceptance as set forth in Article 3.1 hereinabove, and the
+                     Licensee hereby acknowledges that it is aware thereof.</p>
+          
+                </li>
+            </list>
+        </li>
+        <li>
+            <p>Article 4 - EFFECTIVE DATE AND TERM</p>
+            <list>
+                <li>
+                    <b>4.1.</b>
+                    <p>EFFECTIVE DATE</p>
+                    <p>The Agreement shall become effective on the date when it is accepted by the Licensee as set forth
+                     in Article 3.1.</p>
+                </li>
+                <li>
+                    <b>4.2.</b>
+                    <p>TERM</p>
+                    <p>The Agreement shall remain in force during the whole legal term of protection of the economic
+                         rights over the Software.</p>
+                </li>
+            </list>
+        </li>
+        
+        
+        
+        <----------- Brad reviewed through here ----------->
+        
+        
+        
+        
+        <li>
+            <p>Article 5 - SCOPE OF THE RIGHTS GRANTED ---------------------------------------</p>
```

Comment:
extraneous dashes included at the end of this line; don't seem to be present in current canonical source:
http://www.cecill.info/licences/Licence_CeCILL_V1.1-US.html